### DISCUSSION TITLE
[Feature] an installation tutorial in readme.md

### DISCUSSION THREAD

**Discussion by ANightDazingZoroark** [bridge-core/bridge.#131] [Q&A] [answered]
i know this may sound stupid, but this needs to be added somewhere in README.md or something. like i cant wrap my head around the files the releases and how to compile them

**Comment by solvedDev** [comment, marked as answer]
If you want to compile your own builds for bridge. in order to contribute to the project, please follow the [contributing guidelines](https://github.com/solvedDev/bridge./blob/master/CONTRIBUTING.md). 

Installing an already compiled build is as easy as navigating to the releases page (follow the link at the top of the README) and expand the "assets" section of a release.
